packages/backend/src/controllers/users.controller.ts

```
@@ -12,16 +14,78 @@ import { prisma } from '../utils/prisma';
+        },
+      },
     });
     if (!user) return res.status(404).json({ error: 'User not found' });
-    return res.status(200).json(user);
+
+    // Enrich tickets with offer and event info, and mask finalKey
+    const enrichedTickets = await Promise.all(
+      user.tickets.map(async ticket => {
+        const offer = await getOfferById(ticket.offerId);
+        let event = null;
+        if (offer && offer.eventId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)